<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>293</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Pseudorandom number generator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSpinBox" name="value_m_or_p">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>21</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="value_a_or_q">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>61</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="value_c_or_s">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>101</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="value_X0_or_n">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>141</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="m_or_p">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>47</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>m</string>
    </property>
   </widget>
   <widget class="QLabel" name="a_or_q">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>47</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>a</string>
    </property>
   </widget>
   <widget class="QLabel" name="c_or_s">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>47</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>c</string>
    </property>
   </widget>
   <widget class="QLabel" name="X0_or_n">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>47</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>X0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Generate_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>551</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>784</width>
     <height>35</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuType">
    <property name="title">
     <string>Type</string>
    </property>
    <addaction name="actionLehmer"/>
    <addaction name="actionBBS"/>
   </widget>
   <addaction name="menuType"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLehmer">
   <property name="text">
    <string>Lehmer</string>
   </property>
  </action>
  <action name="actionBMS">
   <property name="text">
    <string>BBS</string>
   </property>
  </action>
  <action name="actionBBS">
   <property name="text">
    <string>BBS</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
